Cargo.toml

```
@@ -14,12 +14,19 @@ categories = ["gui"]
 
 license = "MIT OR APACHE-2.0"
 
+[features]
+default = ["x11"]
+x11 = ["xcb"]
+
 [dependencies]
-winit = "0.23.0"
+winit = "0.24.0"
 
-[target.'cfg(windows)'.dependencies]
+[target.'cfg(target_os = "windows")'.dependencies]
 winapi = { version = "0.3.8", features = ["windef", "winuser"] }
 
+[target.'cfg(target_os = "linux")'.dependencies]
```

```suggestion
[target.'cfg(any(target_os = "linux", target_os = "dragonfly", target_os = "freebsd", target_os = "openbsd", target_os = "netbsd"))'.dependencies]
```
I think it's best to not exclude any of the platforms Winit already supports, even if the only one of these platforms I'm even vaguely familiar with is Linux.